src/aleph/http/http2.clj

```diff
@@ -822,6 +822,7 @@
 
      :protocol              "HTTP/2.0"
 
+     :aleph/channel         ch
      :aleph/writable?       writable?
      :aleph/h2-exception    h2-exception
      :aleph/keep-alive?     true                            ; not applicable to HTTP/2, but here for compatibility
```

Comment:
Can you add a comment above that these keys are internal to Aleph, and not to be relied on?

Reply:
Done! Though now I wonder whether `:aleph/request-arrived` actually was intended to be part of the public API since there is no use of it in Aleph itself :thinking:

Reply:
Huh.

Maybe it's for debugging, since nano time has higher resolution than the `Date` header.

Reply:
Looks like Zach meant for the `:aleph/` prefix to distinguish these keys from Ring-defined keys, not to be internal. See https://github.com/clj-commons/aleph/pull/275

We could use two prefixes for each case, `:aleph/` and `:aleph.internal/` to let people know which ones they can rely on.

Reply:
Good call to dig into the history :+1: `:aleph.internal/` sounds good to me, will push a change to that effect momentarily.